### ISSUE TITLE
ci is broken on dev branch

### ISSUE THREAD

**Issue by StStME** [cmake-lint/cmake-lint#28]
currently ci seems to fail at least on dev branch. this should be looked at. @SigureMo : you mentioned you already worked on that on your fork. Is this something which can be taken over as well?

**Comment by SigureMo** [comment]
```
File "/opt/hostedtoolcache/Python/3.7.17/x64/lib/python3.7/site-packages/setuptools/_vendor/importlib_metadata/__init__.py", line 208, in load
    module = import_module(match.group('module'))
  File "/opt/hostedtoolcache/Python/3.7.17/x64/lib/python3.7/importlib/__init__.py", line 1[27](https://github.com/cmake-lint/cmake-lint/actions/runs/8071703537/job/22051885507#step:5:28), in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 7[28](https://github.com/cmake-lint/cmake-lint/actions/runs/8071703537/job/22051885507#step:5:29), in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/hostedtoolcache/Python/3.7.17/x64/lib/python3.7/site-packages/tox_pyenv.py", line 48, in <module>
    from tox import hookimpl as tox_hookimpl
ImportError: cannot import name 'hookimpl' from 'tox' (/opt/hostedtoolcache/Python/3.7.17/x64/lib/python3.7/site-packages/tox/__init__.py)
```

It looks like caused by `tox`, although I don't use `tox` in my fork anymore, but I'll try to fix it.

**Comment by tkruse** [comment]
I tried to use the same setup for cpplint and cmake lint, maybe the cpplint repo had the same issue and already has a fix

**Comment by StStME** [comment]
fixed by #29
